### PR TITLE
Created US13_Muhammad and Pushed

### PR DESCRIPTION
**Pull request by abbi-code**
All good

### CODE REVIEW

**Comment by ozgurSeker80** [comment]
all good

**Review by alcevik — APPROVED** [review]

**Review by ozgurSeker80 — APPROVED** [review]
okay

File: src/test/java/G13/NextBase/tests/US13_Muhammad/AC1.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package G13.NextBase.tests.US13;
```
Comment (abbi-code) [inline src/test/java/G13/NextBase/tests/US13_Muhammad/AC1.java:1]: change this

**Review by abbi-code — COMMENTED** [review]
all good

**Pull request merged**